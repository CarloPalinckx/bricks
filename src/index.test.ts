import initStoryshots, { multiSnapshotWithOptions } from '@storybook/addon-storyshots';
import React, { ReactElement } from 'react';
import ReactDOM from 'react-dom';

// Workaround for https://github.com/facebook/react/issues/11565#issuecomment-368877149
/* tslint:disable */
(ReactDOM as any).createPortal = (node: any): any => React.createElement('MockPortal', null, node);
/* tslint:enable */

jest.mock('./utility/calculateRandomPercentage', (): Function => (): number => 70);

initStoryshots({
    configPath: './config/storybook',
    test: multiSnapshotWithOptions({
        createNodeMock: (element: ReactElement<HTMLDivElement>): Object => ({
            offsetWidth: 900,
            offsetHeight: 900,
            focus: jest.fn(),
            addEventListener: jest.fn(),
            removeEventListener: jest.fn(),
        }),
    }),
});
