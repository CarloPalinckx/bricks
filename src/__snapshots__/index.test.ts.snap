// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots BreakpointProvider default 1`] = `
.c0 {
  background: #eee;
  padding: 24px;
}

<div>
  <div
    className="c0"
  >
    <div>
      Current breakpoint is 
      large
    </div>
  </div>
</div>
`;

exports[`Storyshots Button Destructive 1`] = `
.c0 {
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  border: none;
  line-height: 1;
  padding: 11px 35px;
  box-shadow: 0 6px 3px -5px rgba(51,55,64 ,.3);
  cursor: pointer;
  display: inline-block;
  outline: none;
  -webkit-transform: translateZ(0) translate3d(0,0,0);
  -ms-transform: translateZ(0) translate3d(0,0,0);
  transform: translateZ(0) translate3d(0,0,0);
  -webkit-transition: -webkit-transform 0.1s,background 0.3s,color 0.3s,box-shadow 0.1s;
  -webkit-transition: transform 0.1s,background 0.3s,color 0.3s,box-shadow 0.1s;
  transition: transform 0.1s,background 0.3s,color 0.3s,box-shadow 0.1s;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-family: Source Sans Pro,sans-serif;
  font-size: 16px;
  background-color: #ED2157;
  border-radius: 0;
  border: solid 1px #ED2157;
  color: #fff;
}

.c0:hover {
  background-color: #ED2157;
  border-color: #ED2157;
  color: #fff;
}

.c0:focus {
  background-color: #ED2157;
  border-color: #ED2157;
  color: #fff;
}

.c0:active {
  background-color: #ED2157;
  border-color: #ED2157;
  color: #fff;
}

.c0:hover {
  -webkit-transform: translateY(-2px);
  -ms-transform: translateY(-2px);
  transform: translateY(-2px);
  box-shadow: 0 9px 3px -6px rgba(51,55,64 ,.3);
}

.c0:active {
  -webkit-transform: translateY(0);
  -ms-transform: translateY(0);
  transform: translateY(0);
  box-shadow: 0 2px 3px -2px rgba(51,55,64 ,.3);
}

.c0:focus {
  -webkit-transform: translateY(-2px);
  -ms-transform: translateY(-2px);
  transform: translateY(-2px);
  box-shadow: 0 9px 3px -6px rgba(51,55,64 ,.3);
}

<button
  className="c0"
  href={undefined}
  onClick={[Function]}
  target={undefined}
  title="Click me!"
  type="button"
>
  Click me
</button>
`;

exports[`Storyshots Button Primary 1`] = `
.c0 {
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  border: none;
  line-height: 1;
  padding: 11px 35px;
  box-shadow: 0 6px 3px -5px rgba(51,55,64 ,.3);
  cursor: pointer;
  display: inline-block;
  outline: none;
  -webkit-transform: translateZ(0) translate3d(0,0,0);
  -ms-transform: translateZ(0) translate3d(0,0,0);
  transform: translateZ(0) translate3d(0,0,0);
  -webkit-transition: -webkit-transform 0.1s,background 0.3s,color 0.3s,box-shadow 0.1s;
  -webkit-transition: transform 0.1s,background 0.3s,color 0.3s,box-shadow 0.1s;
  transition: transform 0.1s,background 0.3s,color 0.3s,box-shadow 0.1s;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-family: Source Sans Pro,sans-serif;
  font-size: 16px;
  background-color: #5BD16A;
  border-radius: 0;
  border: solid 1px #5BD16A;
  color: #fff;
}

.c0:hover {
  background-color: #5BD16A;
  border-color: #5BD16A;
  color: #fff;
}

.c0:focus {
  background-color: #5BD16A;
  border-color: #5BD16A;
  color: #fff;
}

.c0:active {
  background-color: #5BD16A;
  border-color: #5BD16A;
  color: #fff;
}

.c0:hover {
  -webkit-transform: translateY(-2px);
  -ms-transform: translateY(-2px);
  transform: translateY(-2px);
  box-shadow: 0 9px 3px -6px rgba(51,55,64 ,.3);
}

.c0:active {
  -webkit-transform: translateY(0);
  -ms-transform: translateY(0);
  transform: translateY(0);
  box-shadow: 0 2px 3px -2px rgba(51,55,64 ,.3);
}

.c0:focus {
  -webkit-transform: translateY(-2px);
  -ms-transform: translateY(-2px);
  transform: translateY(-2px);
  box-shadow: 0 9px 3px -6px rgba(51,55,64 ,.3);
}

<button
  className="c0"
  href={undefined}
  onClick={[Function]}
  target={undefined}
  title="Click me!"
  type="button"
>
  Click me
</button>
`;

exports[`Storyshots Button Secondary 1`] = `
.c0 {
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  border: none;
  line-height: 1;
  padding: 11px 35px;
  box-shadow: 0 6px 3px -5px rgba(51,55,64 ,.3);
  cursor: pointer;
  display: inline-block;
  outline: none;
  -webkit-transform: translateZ(0) translate3d(0,0,0);
  -ms-transform: translateZ(0) translate3d(0,0,0);
  transform: translateZ(0) translate3d(0,0,0);
  -webkit-transition: -webkit-transform 0.1s,background 0.3s,color 0.3s,box-shadow 0.1s;
  -webkit-transition: transform 0.1s,background 0.3s,color 0.3s,box-shadow 0.1s;
  transition: transform 0.1s,background 0.3s,color 0.3s,box-shadow 0.1s;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-family: Source Sans Pro,sans-serif;
  font-size: 16px;
  background-color: #EDEFF2;
  border-radius: 0;
  border: solid 1px #d2d7e0;
  color: #333740;
}

.c0:hover {
  background-color: #EDEFF2;
  border-color: #d2d7e0;
  color: #333740;
}

.c0:focus {
  background-color: #EDEFF2;
  border-color: #d2d7e0;
  color: #333740;
}

.c0:active {
  background-color: #EDEFF2;
  border-color: #d2d7e0;
  color: #333740;
}

.c0:hover {
  -webkit-transform: translateY(-2px);
  -ms-transform: translateY(-2px);
  transform: translateY(-2px);
  box-shadow: 0 9px 3px -6px rgba(51,55,64 ,.3);
}

.c0:active {
  -webkit-transform: translateY(0);
  -ms-transform: translateY(0);
  transform: translateY(0);
  box-shadow: 0 2px 3px -2px rgba(51,55,64 ,.3);
}

.c0:focus {
  -webkit-transform: translateY(-2px);
  -ms-transform: translateY(-2px);
  transform: translateY(-2px);
  box-shadow: 0 9px 3px -6px rgba(51,55,64 ,.3);
}

<button
  className="c0"
  href={undefined}
  onClick={[Function]}
  target={undefined}
  title="Click me!"
  type="button"
>
  Click me
</button>
`;

exports[`Storyshots Button Warning 1`] = `
.c0 {
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  border: none;
  line-height: 1;
  padding: 11px 35px;
  box-shadow: 0 6px 3px -5px rgba(51,55,64 ,.3);
  cursor: pointer;
  display: inline-block;
  outline: none;
  -webkit-transform: translateZ(0) translate3d(0,0,0);
  -ms-transform: translateZ(0) translate3d(0,0,0);
  transform: translateZ(0) translate3d(0,0,0);
  -webkit-transition: -webkit-transform 0.1s,background 0.3s,color 0.3s,box-shadow 0.1s;
  -webkit-transition: transform 0.1s,background 0.3s,color 0.3s,box-shadow 0.1s;
  transition: transform 0.1s,background 0.3s,color 0.3s,box-shadow 0.1s;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-family: Source Sans Pro,sans-serif;
  font-size: 16px;
  background-color: #FCC200;
  border-radius: 0;
  border: solid 1px #FCC200;
  color: #fff;
}

.c0:hover {
  background-color: #FCC200;
  border-color: #FCC200;
  color: #fff;
}

.c0:focus {
  background-color: #FCC200;
  border-color: #FCC200;
  color: #fff;
}

.c0:active {
  background-color: #FCC200;
  border-color: #FCC200;
  color: #fff;
}

.c0:hover {
  -webkit-transform: translateY(-2px);
  -ms-transform: translateY(-2px);
  transform: translateY(-2px);
  box-shadow: 0 9px 3px -6px rgba(51,55,64 ,.3);
}

.c0:active {
  -webkit-transform: translateY(0);
  -ms-transform: translateY(0);
  transform: translateY(0);
  box-shadow: 0 2px 3px -2px rgba(51,55,64 ,.3);
}

.c0:focus {
  -webkit-transform: translateY(-2px);
  -ms-transform: translateY(-2px);
  transform: translateY(-2px);
  box-shadow: 0 9px 3px -6px rgba(51,55,64 ,.3);
}

<button
  className="c0"
  href={undefined}
  onClick={[Function]}
  target={undefined}
  title="Click me!"
  type="button"
>
  Click me
</button>
`;

exports[`Storyshots Button With a link 1`] = `
.c0 {
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  border: none;
  line-height: 1;
  padding: 11px 35px;
  box-shadow: 0 6px 3px -5px rgba(51,55,64 ,.3);
  cursor: pointer;
  display: inline-block;
  outline: none;
  -webkit-transform: translateZ(0) translate3d(0,0,0);
  -ms-transform: translateZ(0) translate3d(0,0,0);
  transform: translateZ(0) translate3d(0,0,0);
  -webkit-transition: -webkit-transform 0.1s,background 0.3s,color 0.3s,box-shadow 0.1s;
  -webkit-transition: transform 0.1s,background 0.3s,color 0.3s,box-shadow 0.1s;
  transition: transform 0.1s,background 0.3s,color 0.3s,box-shadow 0.1s;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-family: Source Sans Pro,sans-serif;
  font-size: 16px;
  background-color: #5BD16A;
  border-radius: 0;
  border: solid 1px #5BD16A;
  color: #fff;
}

.c0:hover {
  background-color: #5BD16A;
  border-color: #5BD16A;
  color: #fff;
}

.c0:focus {
  background-color: #5BD16A;
  border-color: #5BD16A;
  color: #fff;
}

.c0:active {
  background-color: #5BD16A;
  border-color: #5BD16A;
  color: #fff;
}

.c0:hover {
  -webkit-transform: translateY(-2px);
  -ms-transform: translateY(-2px);
  transform: translateY(-2px);
  box-shadow: 0 9px 3px -6px rgba(51,55,64 ,.3);
}

.c0:active {
  -webkit-transform: translateY(0);
  -ms-transform: translateY(0);
  transform: translateY(0);
  box-shadow: 0 2px 3px -2px rgba(51,55,64 ,.3);
}

.c0:focus {
  -webkit-transform: translateY(-2px);
  -ms-transform: translateY(-2px);
  transform: translateY(-2px);
  box-shadow: 0 9px 3px -6px rgba(51,55,64 ,.3);
}

<a
  className="c0"
  href="http://www.google.nl"
  onClick={[Function]}
  target="_blank"
  title="Click me!"
  type={undefined}
>
  Click me
</a>
`;

exports[`Storyshots FlatButton default 1`] = `
.c0 {
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  border: none;
  line-height: 1;
  padding: 11px 35px;
  box-shadow: none;
  cursor: pointer;
  display: inline-block;
  outline: none;
  -webkit-transform: translateZ(0) translate3d(0,0,0);
  -ms-transform: translateZ(0) translate3d(0,0,0);
  transform: translateZ(0) translate3d(0,0,0);
  -webkit-transition: -webkit-transform 0.1s,background 0.3s,color 0.3s,box-shadow 0.1s;
  -webkit-transition: transform 0.1s,background 0.3s,color 0.3s,box-shadow 0.1s;
  transition: transform 0.1s,background 0.3s,color 0.3s,box-shadow 0.1s;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  -webkit-text-decoration: underline;
  text-decoration: underline;
  font-family: Source Sans Pro,sans-serif;
  font-size: 16px;
  background: transparent;
  border-radius: 0;
  border: solid 1px transparent;
  color: #21252B;
}

.c0:hover {
  background-color: transparent;
  border-color: #d2d7e0;
  color: #21252B;
  -webkit-transform: translateY(-2px);
  -ms-transform: translateY(-2px);
  transform: translateY(-2px);
  box-shadow: 0 9px 3px -6px rgba(51,55,64 ,.3);
}

.c0:focus {
  background-color: transparent;
  border-color: #d2d7e0;
  color: #21252B;
  -webkit-transform: translateY(-2px);
  -ms-transform: translateY(-2px);
  transform: translateY(-2px);
  box-shadow: 0 9px 3px -6px rgba(51,55,64 ,.3);
}

.c0:active {
  background-color: transparent;
  border-color: #d2d7e0;
  color: #21252B;
  -webkit-transform: translateY(0);
  -ms-transform: translateY(0);
  transform: translateY(0);
  box-shadow: 0 2px 3px -2px rgba(51,55,64 ,.3);
}

<button
  className="c0"
  href={undefined}
  onClick={[Function]}
  target={undefined}
  title="Click me!"
  type="button"
>
  Click me
</button>
`;

exports[`Storyshots FoldOut With a toggle 1`] = `
.c1 {
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  border: none;
  line-height: 1;
  padding: 11px 35px;
  box-shadow: 0 6px 3px -5px rgba(51,55,64 ,.3);
  cursor: pointer;
  display: inline-block;
  outline: none;
  -webkit-transform: translateZ(0) translate3d(0,0,0);
  -ms-transform: translateZ(0) translate3d(0,0,0);
  transform: translateZ(0) translate3d(0,0,0);
  -webkit-transition: -webkit-transform 0.1s,background 0.3s,color 0.3s,box-shadow 0.1s;
  -webkit-transition: transform 0.1s,background 0.3s,color 0.3s,box-shadow 0.1s;
  transition: transform 0.1s,background 0.3s,color 0.3s,box-shadow 0.1s;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-family: Source Sans Pro,sans-serif;
  font-size: 16px;
  background-color: #EDEFF2;
  border-radius: 0;
  border: solid 1px #d2d7e0;
  color: #333740;
}

.c1:hover {
  background-color: #EDEFF2;
  border-color: #d2d7e0;
  color: #333740;
}

.c1:focus {
  background-color: #EDEFF2;
  border-color: #d2d7e0;
  color: #333740;
}

.c1:active {
  background-color: #EDEFF2;
  border-color: #d2d7e0;
  color: #333740;
}

.c1:hover {
  -webkit-transform: translateY(-2px);
  -ms-transform: translateY(-2px);
  transform: translateY(-2px);
  box-shadow: 0 9px 3px -6px rgba(51,55,64 ,.3);
}

.c1:active {
  -webkit-transform: translateY(0);
  -ms-transform: translateY(0);
  transform: translateY(0);
  box-shadow: 0 2px 3px -2px rgba(51,55,64 ,.3);
}

.c1:focus {
  -webkit-transform: translateY(-2px);
  -ms-transform: translateY(-2px);
  transform: translateY(-2px);
  box-shadow: 0 9px 3px -6px rgba(51,55,64 ,.3);
}

.c2 {
  -webkit-transition: height 300ms cubic-bezier(0.5,0,0.1,1);
  transition: height 300ms cubic-bezier(0.5,0,0.1,1);
  overflow: hidden;
  height: 0;
}

.c0 {
  margin: 0px 0px 24px 0px;
}

<div>
  <div
    className="c0"
  >
    <button
      className="c1"
      href={undefined}
      onClick={[Function]}
      target={undefined}
      title="open FoldOut"
      type="button"
    >
      Toggle
    </button>
  </div>
  <div
    className="c2"
  >
    <div>
      
    Now this is the story all about how
    My life got flipped, turned upside down
    And I'd like to take a minute just sit right there
    I'll tell you how I became the prince of a town called Bel-air

    In west Philadelphia born and raised
    On the playground where I spent most of my days
    Chilling out, maxing, relaxing all cool
    And all shooting some b-ball outside of the school
    When a couple of guys, they were up to no good
    Started making trouble in my neighbourhood
    I got in one little fight and my mom got scared
    And said "You're moving with your auntie and uncle in Bel-air"

    I whistled for a cab and when it came near the
    License plate said "fresh" and had a dice in the mirror
    If anything I could say that this cab was rare
    But I thought nah, forget it, yo homes to Bel-air!

    I pulled up to a house about seven or eight
    And I yelled to the cabby "Yo, homes smell you later!"
    Looked at my kingdom I was finally there
    To sit on my throne as the prince of Bel-air

    </div>
  </div>
</div>
`;

exports[`Storyshots Heading With a hierarchy 1`] = `
.c0 {
  color: #333740;
  font-family: Melbourne,sans-serif;
  font-size: 28px;
  font-weight: 400;
  line-height: 1.5;
  margin: 0;
}

<h1
  className="c0"
  role="heading"
>
  This is an h
  1
   element.
</h1>
`;

exports[`Storyshots Heading default 1`] = `
.c0 {
  color: #333740;
  font-family: Melbourne,sans-serif;
  font-size: 28px;
  font-weight: 400;
  line-height: 1.5;
  margin: 0;
}

<div
  className="c0"
  role="heading"
>
  This is a div element.
</div>
`;

exports[`Storyshots Icon Large icon 1`] = `
.c0 {
  display: inline-block;
  vertical-align: middle;
  height: 60px;
  width: 60px;
}

.c0 svg {
  max-height: 100%;
  max-width: 100%;
  fill: currentColor;
}

<span
  aria-hidden={true}
  className="c0"
  dangerouslySetInnerHTML={
    Object {
      "__html": "test-file-stub",
    }
  }
  role="img"
/>
`;

exports[`Storyshots Icon Small icon 1`] = `
.c0 {
  display: inline-block;
  vertical-align: middle;
  height: 18px;
  width: 18px;
}

.c0 svg {
  max-height: 100%;
  max-width: 100%;
  fill: currentColor;
}

<span
  aria-hidden={true}
  className="c0"
  dangerouslySetInnerHTML={
    Object {
      "__html": "test-file-stub",
    }
  }
  role="img"
/>
`;

exports[`Storyshots Icon With overwritten fill 1`] = `
.c0 {
  display: inline-block;
  vertical-align: middle;
  height: 60px;
  width: 60px;
}

.c0 svg {
  max-height: 100%;
  max-width: 100%;
  fill: BlueViolet;
}

<span
  aria-hidden={true}
  className="c0"
  dangerouslySetInnerHTML={
    Object {
      "__html": "test-file-stub",
    }
  }
  role="img"
/>
`;

exports[`Storyshots Icon With some text 1`] = `
.c1 {
  margin: 12px 0px 0px 0px;
}

.c0 {
  display: inline-block;
  vertical-align: middle;
  height: 18px;
  width: 18px;
}

.c0 svg {
  max-height: 100%;
  max-width: 100%;
  fill: currentColor;
}

.c2 {
  display: inline-block;
  vertical-align: middle;
  height: 60px;
  width: 60px;
}

.c2 svg {
  max-height: 100%;
  max-width: 100%;
  fill: currentColor;
}

<div>
  <span
    aria-hidden={true}
    className="c0"
    dangerouslySetInnerHTML={
      Object {
        "__html": "test-file-stub",
      }
    }
    role="img"
  />
  <span>
     With some text
  </span>
  <div
    className="c1"
  >
    <span
      aria-hidden={true}
      className="c2"
      dangerouslySetInnerHTML={
        Object {
          "__html": "test-file-stub",
        }
      }
      role="img"
    />
    <span>
       With some other text
    </span>
  </div>
</div>
`;

exports[`Storyshots Spacer With margin 1`] = `
.c0 {
  margin: 12px 12px 12px 12px;
}

.c1 {
  border: solid 3px rgba(255,36,94,0.3);
  border-radius: 5px;
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    This has a margin.
  </div>
</div>
`;

exports[`Storyshots Spacer With padding 1`] = `
.c1 {
  padding: 12px 12px 12px 12px;
}

.c0 {
  border: solid 3px rgba(255,36,94,0.3);
  border-radius: 5px;
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    This has padding.
  </div>
</div>
`;

exports[`Storyshots Spacer Without props 1`] = `
.c0 {
  margin: 0px 0px 0px 0px;
}

.c1 {
  border: solid 3px rgba(255,36,94,0.3);
  border-radius: 5px;
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    This has no spacing.
  </div>
</div>
`;

exports[`Storyshots SubHeading With a hierarchy 1`] = `
.c0 {
  color: #88979d;
  font-family: Source Sans Pro,sans-serif;
  font-size: 18px;
  font-weight: 400;
  line-height: 1.5;
  margin: 0;
}

<h2
  className="c0"
  role="heading"
>
  This is an h
  2
   element.
</h2>
`;

exports[`Storyshots SubHeading default 1`] = `
.c0 {
  color: #88979d;
  font-family: Source Sans Pro,sans-serif;
  font-size: 18px;
  font-weight: 400;
  line-height: 1.5;
  margin: 0;
}

<div
  className="c0"
  role="heading"
>
  This is a div element.
</div>
`;

exports[`Storyshots Text default 1`] = `
.c0 {
  color: #333740;
  font-family: Source Sans Pro,sans-serif;
  font-size: 16px;
  font-weight: 400;
  line-height: 1.5;
  margin: 0;
}

<p
  className="c0"
>
  
    Now this is the story all about how
    My life got flipped, turned upside down
    And I'd like to take a minute just sit right there
    I'll tell you how I became the prince of a town called Bel-air

    In west Philadelphia born and raised
    On the playground where I spent most of my days
    Chilling out, maxing, relaxing all cool
    And all shooting some b-ball outside of the school
    When a couple of guys, they were up to no good
    Started making trouble in my neighbourhood
    I got in one little fight and my mom got scared
    And said "You're moving with your auntie and uncle in Bel-air"

    I whistled for a cab and when it came near the
    License plate said "fresh" and had a dice in the mirror
    If anything I could say that this cab was rare
    But I thought nah, forget it, yo homes to Bel-air!

    I pulled up to a house about seven or eight
    And I yelled to the cabby "Yo, homes smell you later!"
    Looked at my kingdom I was finally there
    To sit on my throne as the prince of Bel-air

</p>
`;
